### DISCUSSION TITLE
Anilist - can't authorize account

### DISCUSSION THREAD

**Discussion by Jim22223** [SwagOtaku/plugin.video.kaito#41] [Q&A] [answered]
I am trying to add an Anilist account. After entering the Username and Auth Token both the "Authorize" and "Cancel" buttons don't work.

**Comment by icpryde** [comment, marked as answer]
are you doing this on android or windows? I do notice the authorize button can be kinda "stuck" if you can get a mouse to click authorize that is the best option.

**Comment by SwagOtaku** [comment]
@Jim22223 were you able to authorize?

**Reply by Jim22223** [reply]
hi,

sorry. I didn't see the reply. Yes, I managed it. I simply hit enter on the keyboard. I am on Windows 10. The plugin works really well.

**Reply by SwagOtaku** [reply]
no problem, glad it worked for you
